assemble/conf/accumulo-env.sh

```diff
@@ -144,3 +144,6 @@ esac
 
 ## Specifies command that will be placed before calls to Java in accumulo script
 # export ACCUMULO_JAVA_PREFIX=""
+# export ACCUMULO_BIND_ADDR=""
+# export ACCUMULO_COMPACTOR_QUEUE=""
+# export ACCUMULO_SSERVER_GROUP=""
```

Comment:
We should not be adding new environment variables. One of the design goals of the scripts starting in 2.0 was to be less tightly coupled to environment variables baked in to our startup scripts. These lock users in to features of `bin/accumulo` and makes `bin/accumulo` do more than merely "run java main class with provided args".

Rather than add these environment variables, and then bake in the options into the `bin/accumulo` script, users should just specify `-o <whatever>` themselves, as needed. We do not need to, and should not, be selectively picking a subset of our configuration options, and providing yet-another-way to configure those same options. This creates confusion and makes it hard for users to know the "right" or "best" way to configure things.

Reply:
all changes have been removed from accumulo-env.sh